### PR TITLE
Update TCJA references to reconciliation version

### PR DESCRIPTION
**Pull request by MaxGhenis**
The reconciliation TCJA:
1) Raises _DependentCredit_Nonchild_c to $500, not $300 (https://github.com/open-source-economics/Tax-Calculator/blob/master/taxcalc/reforms/TCJA_Reconciliation.json#L131)
2) Does away with _FilerCredit_c

### CODE REVIEW

File: taxcalc/current_law_policy.json [hunk 1]
```diff
@@ -3679,7 +3679,6 @@
         "section_1": "Nonrefundable Credits",
         "section_2": "Child Tax Credit",
         "irs_ref": "",
-        "notes": "The Tax Cuts and Jobs Act sets this at $600 for joint and $300 for others",
```
Comment (martinholmer) [inline taxcalc/current_law_policy.json]: @MaxGhenis,  Can't remove the "notes", but can set to this: `"notes": "",`

**Comment by martinholmer** [comment]
@codykallen,  Can you review PR #1765?  
In that PR @MaxGhenis is suggesting some updates to the "notes" for the `_DependentCredit_Nonchild_c` and `_FilerCredit_c` policy parameters that you added in early November as part of #1615 and #1618, respectively.

File: taxcalc/current_law_policy.json (hunk 1 above)
Comment (MaxGhenis) [reply taxcalc/current_law_policy.json]: Oops, fixed.

Comment (martinholmer) [reply taxcalc/current_law_policy.json]: Thanks @MaxGhenis.  I've been pretty much out of the loop on the details of specifying the iterations of the TCJA reforms, so I think it is best to wait for Cody's review.  Thanks for your careful review of our work.

**Comment by MaxGhenis** [comment]
Sure, it might make more sense to note either all TCJA changes or none, but
thought I'd make the minimal change for the previously noted ones.

**Comment by MattHJensen** [comment]
The primary value of having these TCJA references in `current_law_policy.json` -- unless I'm missing something -- is that they give the user a contextual reference for the parameters that don't appear in current law. I don't think it would undermine this objective to remove the dollar amount and only cite the bill ("TCJA") in the notes section. It would also reduce the maintenance load.

**Comment by martinholmer** [comment]
@MattHJensen said:
> The primary value of having these TCJA references in `current_law_policy.json` -- unless I'm missing something -- is that they give the user a contextual reference for the parameters that don't appear in current law. I don't think it would undermine this objective to remove the dollar amount and only cite the bill ("TCJA") in the notes section. It would also reduce the maintenance load.

@MaxGhenis,  These are important points.  How about dropping the dollar amounts and rewording the remaining "notes" fields to be something like `First introduced in TCJA bills`?  Saying these parameters are not part of current-law policy is risky because the probability is high that they will be part of current-law policy in a matter of days.

**Comment by MaxGhenis** [comment]
> How about dropping the dollar amounts and rewording the remaining "notes" fields to be something like First introduced in TCJA bills?

Done, for the two that currently reference TCJA. 

I also filed #1777 as a longer-term feature request to understand the origin and evolution of each parameter.

**Comment by martinholmer** [comment]
Thanks @MaxGhenis for the enhancement in pull request #1765.

**Pull request merged**